### PR TITLE
New .MD file for Bhagyesh

### PR DESCRIPTION
**Pull request by bhagyesh12**
Added some places to go #41 @gargvk81

### CODE REVIEW

File: bhagyesh12_NYC.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# New York, NY
```
Comment (pkolakaluri) [inline bhagyesh12_NYC.md:1]: Why not Downtown

File: bhagyesh12_NYC.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
...
+
+### Places to go
+
+Empire State Building
+Times Square
+Status of Liberty
+
```
Comment (coderdecoder01) [inline bhagyesh12_NYC.md]: How's the weather like ?

Comment (bhagyesh12) [reply bhagyesh12_NYC.md]: Cold

File: bhagyesh12_NYC.md (hunk 1 above)
Comment (bhagyesh12) [reply bhagyesh12_NYC.md:1]: Dont like Downtown, we work there

**Pull request merged**